docs/Learning-Environment-Create-New.md

```diff
@@ -529,28 +491,73 @@ the Python API work as expected using the `notebooks/getting-started.ipynb`
 `env_name` to the name of the environment file you specify when building this
 environment.
 
+## Training the Environment
+
 Now you can train the Agent. To get ready for training, you must first to change
 the `Brain` of the agent to be the Learning Brain `RollerBallBrain`.
-Then drag the `RollerBallBrain` into the Academy's `Broadcast Hub` and check 
-to `Control` checkbox for that brain. From there, the process is
+Then, select the Academy GameObject and check the `Control` checkbox for 
+the RollerBallBrain item in the **Broadcast Hub** list. From there, the process is
 the same as described in [Training ML-Agents](Training-ML-Agents.md).
 
+The hyperparameters for training are specified in the configuration file that you ls
+pass to the `mlagents-learn` program. Using the default settings specified 
+in the `config/trainer_config.yaml` file (in your ml-agents folder), the
+RollerAgent takes about 300,000 steps to train. However, you can change the 
+following hyperparameters  to speed up training considerably (to under 20,000 steps):
+
+    batch_size: 1
```

Comment:
Are you sure these hyperparameters work better than the defaults for balance ball ?

Reply:
150x better!

Reply:
Have you tried with batch size 10 and buffer size 100. What are the combinations of hyperparameters you used ?

Reply:
Yes, I basically walked batch/buffer size down by halves from the default 1024/10240 to arrive at the current recommendation of 1/10. 10/100 was better than higher values, but 1/10 was better still. I didn't find any other hyperparameters had much in any effect on training, though I tested the others in isolation, not in combination. (A higher Decision Frequency setting also improved training, but that isn't technically a hyperparameter in ml-agents nomenclature.)

Reply:
How much slower is it? I really don't think it is a good idea to use hyperparameters that are not in the recommended range.

Reply:
After increasing the Decision Frequency, the difference isn't as great as I previously noted. I did several runs with each set of values. Best case, 1/10 was about 10k steps faster than 10/100, but was also more variable, with a worse case around 50k steps, which was almost twice the average 10/100 run. I took your advice and changed the recommended values to 10/100.